src/fides/api/alembic/migrations/versions/1e750936a141_migrate_to_fideslang_2_0_uses_and_.py

```diff
@@ -230,10 +247,11 @@ def update_ctl_policy_data_categories(
         for i, rule in enumerate(rules or []):
             data_labels: Dict = rule.get("data_categories", {})
             for j, val in enumerate(data_labels.get("values", [])):
-                new_data_label: Optional[str] = data_label_map.get(val, None)
-                if new_data_label:
-                    rules[i]["data_categories"]["values"][j] = new_data_label
-                    needs_update = True
```

Comment:
Looks like you dropped the `needs_update=True` with this change, so data category updates to ctl_policies will no longer work

Reply:
I dropped it but set it to update all regardless

Reply:
that's not true as of your latest commit, it is still using a `needs_update` flag -

Reply:
fixed